view_post.php

```diff
@@ -1,3 +1,7 @@
+<?php
+include ('/Applications/MAMP/htdocs/include/PHPProject.php');
```

Comment:
I think you should be able to write this:

```
include ('include/PHPProject.php');
```

Your goal should be for your code to only be aware of what's inside the /htdocs folder. Anything outside that folder might be different on the live server, or if you try to run this on a different computer.